org.coolero.Coolero.yaml

```diff
@@ -0,0 +1,117 @@
+app-id: org.coolero.Coolero
+runtime: org.kde.Platform
+runtime-version: '6.2'
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm12
+command: coolero.sh
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=all
+  - --device=dri
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=org.freedesktop.Flatpak
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - '*.la'
+  - '*.a'
+modules:
+  - name: libusb
```

Comment:
you could use the one from shared-modules.

Reply:
sure, done.